source/mapKeys.js

```diff
@@ -0,0 +1,23 @@
+import _curry2 from './internal/_curry2.js';
+import _mapKeys from './internal/_mapKeys.js';
+
+/**
+ * Transforms an object by converting the keys to new values.
+ *
+ * **Note** that if multiple keys map to the same new key, the last one processed will dominate.
+ *
+ * @func
+ * @memberOf R
+ * @category Object
+ * @sig (String -> String) -> Object -> Object
+ * @param {Function} fn
+ * @param {Object} obj
+ * @return {Object}
+ * @see R.map
```

Comment:
Can `@see` the other new functions here as well

Reply:
yes, just forgot.  Cut and pasted the JSDoc stuff from elsewhere.  Will try to fix tomorrow.